javascript/javascript1/week1/smart-ease.js

```diff
@@ -0,0 +1,87 @@
+//FreeCOdeCamp Exercise link
+//https://www.freecodecamp.org/myselfankitaa
+
+//Age-ify (A future age calculator)
+const yearOfBirth = 1988;
+let yearFuture = 2027;
+const age = yearFuture - yearOfBirth;
+console.log("You will be " + age + " years old in " + yearFuture);
+
+//Goodboy-Oldboy (A dog age calculator)
+const dogYearOfBirth = 2020;
+let dogYearFuture = 2045;
+const dogAge = dogYearFuture - dogYearOfBirth;
+let dogYear = dogAge * 7;
+let shouldShowResultInDogYears = true;
+if (shouldShowResultInDogYears) {
+  console.log(
+    "Your dog will be " + dogYear + " dog years old in " + dogYearFuture
+  );
+} else {
+  console.log(
+    "Your dog will be " + dogAge + " human years old in " + dogYearFuture
+  );
+}
+
+//Housey pricey (A house price estimator)
+const peterHouseWidth = 8;
+const peterHouseDepth = 10;
+const peterHouseheight = 10;
+const peterGardenSizeInM2 = 100;
+const peterHouseActualPrice = 2500000;
+let volumeInMeters = peterHouseheight * peterHouseDepth * peterHouseWidth;
+let housePrice = volumeInMeters * 2.5 * 1000 + peterGardenSizeInM2 * 300;
+console.log("Actual house Price is " + housePrice);
+if (housePrice > peterHouseActualPrice) {
+  console.log("Peter paid too little.");
+} else {
+  console.log("Peter paid too much.");
+}
+
+const juliaHouseWidth = 5;
+const juliaHouseDepth = 11;
+const juliaHouseheight = 8;
+const juliaGardenSizeInM2 = 70;
+const juliaHouseActualPrice = 1000000;
+volumeInMeters = juliaHouseheight * juliaHouseDepth * juliaHouseWidth;
+housePrice = volumeInMeters * 2.5 * 1000 + juliaGardenSizeInM2 * 300;
```

Comment:
I can recommend separating concerns as much as possible. Instead of re-assigning the original variables `volumeInMeters` I would split them into separate variables:

I.e. `peterVolumeInMeters` and `juliaVolumeInMeters`

Your code will work but it is an unnecessary re-assigning.

Reply:
Thanks, I understood and made the changes. It is really helping to make variables clear in the code.